src/java.desktop/share/classes/javax/swing/plaf/basic/BasicLookAndFeel.java

```diff
@@ -2190,11 +2180,10 @@ static void playSound(JComponent c, Object actionKey) {
      * This class contains listener that watches for all the mouse
      * events that can possibly invoke popup on the component
      */
-    class AWTEventHelper implements AWTEventListener,PrivilegedAction<Object> {
-        @SuppressWarnings("removal")
+    class AWTEventHelper implements AWTEventListener {
         AWTEventHelper() {
             super();
-            AccessController.doPrivileged(this);
+            run();
```

Comment:
I don't think this method being called `run()` still makes sense when just being called from the constructor, it used to make (more) sense when it was an override from PrivilegedAction.

Reply:
I thought about that, but I was limiting the changes.